examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,70 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+$ oc cluster up
+
+If you do not use oc cluster up, ensure the imagestreams are registered in the openshift namespace, as well as the
+jenkins template represented by jenkinstemplate.json:
+
```

Comment:
i think you should do `oc login -u system:admin` before importing the templates since after `oc cluster up` you are user developer